core/templates/components/entity-creation-services/story-creation.service.spec.ts

```diff
@@ -0,0 +1,144 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Modal and functionality for the create story button.
+ */
+
+require(
+  'pages/topic-editor-page/modal-templates/' +
+  'create-new-story-modal.controller.ts');
+
+require('domain/utilities/url-interpolation.service.ts');
+require('pages/topic-editor-page/services/topic-editor-state.service.ts');
+require('services/alerts.service.ts');
+
+import { HttpClientTestingModule, HttpTestingController }
+  from '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks }
+  from '@angular/core/testing';
+
+import { AlertsService } from 'services/alerts.service';
+import { LoaderService } from 'services/loader.service';
+import { importAllAngularServices } from 'tests/unit-test-utils';
+
+describe('Collection Creation service', () => {
```

Comment:
Is this Collection creation service?

Reply:
No its a story-creation.service
i will remove that